behave/features/environment.py

```diff
@@ -1,26 +1,32 @@
 import os
 import time
+import logging
+import subprocess
 from behave.contrib.scenario_autoretry import patch_scenario_with_autoretry
 from behave.log_capture import capture
+
 from helper.constants import (
     BROWSER,
-    ARTIFACTS_DIRECTORY,
-    DOWNLOAD_DIRECTORY,
+    DATA_DIRECTORY,
     A11Y_TAG,
+    DATABASE_SNAPSHOT_ENABLED,
 )
+
 from helper.helper_web import get_browser
 from features.steps.common_steps import check_accessibility, make_dir, get_tag
-import logging
 
 
 def before_all(context):
+    context.feature_errors_dir = os.path.join(DATA_DIRECTORY, "feature_errors")
+    make_dir(context.feature_errors_dir)
+    context.download_dir = os.path.join(DATA_DIRECTORY, "downloads")
+    make_dir(context.download_dir)
+    context.a11y_reports_dir = os.path.join(DATA_DIRECTORY, "a11y_reports")
```

Comment:
Ah okay, I think I've mislead you. A11y reports go into feature_errrors, it doesn't need it's own directory, unless what you're changing requires it?

The directory creation already exisits in common_steps. I've added a comment where I was originally thinking sorry.